### PR TITLE
updated links

### PR DESCRIPTION
**Pull request by puttyq**
- added Mastering Powershell Book reference
- added posh-ssh cmdline link
- added ps cheat sheet link

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -88,6 +89,7 @@ It includes a command-line shell and an associated scripting language.
 ## Commandline Productivity
 
 * [posh-git](https://github.com/dahlbyk/posh-git) - Set of PowerShell scripts which provide Git/PowerShell integration.
+* [posh-ssh](https://github.com/darkoperator/Posh-SSH) - PowerShell Module for automating tasks on remote systems using SSH
```
Comment (janikvonrotz) [inline readme.md]: Win10 supports ssh.

File: readme.md [hunk 2]
```diff
@@ -57,6 +57,7 @@ It includes a command-line shell and an associated scripting language.
 
 ## Books
 
+* [Mastering PowerShell](https://www.bonusbits.com/uploads/Mastering-PowerShell.pdf) - a free eBook that covers everything required to become self-sufficient using PowerShell
```
Comment (janikvonrotz) [inline readme.md]: Suitable addition.

**Comment by janikvonrotz** [comment]
PR requires update.

File: readme.md (hunk 1 above)
Comment (puttyq) [reply readme.md]: Although Win10 supports SSH, posh-ssh can be used on any .Net4.5 and PowerShell v3+ machine. We used it on a bunch of Windows Server 2008 R2 - thus it was suggested. Would you rather like to remove?

**Pull request closed without merging**